cmd/csi-api-gen/generators/api_group_generated_gen.go

```diff
@@ -0,0 +1,74 @@
+package generators
```

Comment:
is this file auto generated?

Reply:
No. It contains the generator that generates `api_group_generated.go` files, hence the name.